ios/FluentUI.Demo/FluentUI.Demo.xcodeproj/xcshareddata/xcschemes/Demo.Release.xcscheme

```diff
@@ -90,6 +90,17 @@
                ReferencedContainer = "container:FluentUI.Demo.xcodeproj">
             </BuildableReference>
          </TestableReference>
+         <TestableReference
```

Comment:
are you out of sync? didn't @joannaqu recently remove?

Reply:
#1537 ? If so, it was merged this morning and isn't present on this branch. I can rebase this branch with it.

Reply:
Yep, Mike merged it in this morning. Rebasing will also remove the pr check that's not running right now.